src/platform/user/authentication/utilities.js

```diff
@@ -14,25 +14,30 @@ export const ssoKeepAliveEndpoint = () => {
   return `https://${envPrefix}eauth.va.gov/keepalive`;
 };
 
-function sessionTypeUrl(type = '', version = 'v0', application = null) {
+function sessionTypeUrl(
+  type = '',
+  version = 'v0',
+  application = null,
+  to = null,
+) {
   const SESSIONS_URI =
     version === 'v1'
       ? `${environment.API_URL}/v1/sessions`
       : `${environment.API_URL}/sessions`;
 
   return `${SESSIONS_URI}/${type}/new${
     application ? `?application=${application}` : ''
-  }`;
+  }${application && to ? `&to=${to}` : ''}`;
```

Comment:
this is starting to look a little ugly. I'm gonna refactor this into using `URLSearchParams` so it's more readable

Reply:
also, for the purposes of unit testing: is it the proper intent that we disregard `to` params unless they're paired with an `application` param, too?

Reply:
yeah I was not proud of that particular line of code

Reply:
merge in master, I updated this logic in https://github.com/department-of-veterans-affairs/vets-website/pull/12623